examples/trigger_soundfile.rs

```diff
@@ -8,7 +8,7 @@ fn main() {
 
     // load and decode buffer
     let file = File::open("samples/sample.wav").unwrap();
-    let audio_buffer = context.decode_audio_data(file).unwrap();
+    let audio_buffer = context.decode_audio_data_sync(file).unwrap();
 
     // @fixme - if only one node in the graph it is never removed even when returning
```

Comment:
absolutely not related, but this should be rechecked that was kind of strange

Reply:
I will try to figure out how to make a test for that, that's something I've seen with dirty `println!`

Reply:
Thanks, very odd indeed and something we should investigate